_posts/2021-07-28-how-to-update-to-cis-14.adoc

```diff
@@ -0,0 +1,349 @@
+with v1.4.0 of the CIS AWS Foundations Benchmark. We strongly recommend upgrading straight to at least v0.27.0,
+instead of each minor version incrementally. This saves a lot of time due to the performance enhancements in
+recent versions of the account baseline modules and you can still complete the required migration steps before
+running Terraform/Terragrunt `apply`.
+
+NOTE: This guide assumes you are using at least v0.22.0 of the CIS Service Catalog repo.
+
+The account baseline modules had three breaking changes between versions v0.22.0 and v0.27.0. We must manually run
+these migration steps before updating the module versions.
+
+1. https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/releases/tag/v0.23.0[v0.23.0]: Refactored the
+SecurityHub module to remove a Python script that managed invitations between the AWS accounts. It's necessary to run a
+state migration to manage the invitations with Terraform.
+2. https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/releases/tag/v0.24.0[v0.24.0]: This release introduces MFA Delete. You will need to follow the migration guide to ensure all S3 buckets are properly secured. Note: It is unlikely you will need to perform this step on the AWS root account as they typically don't contain S3 buckets. Please ensure you migrate all other AWS accounts.
+3. https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/releases/tag/v0.25.0[v0.25.0]: Update the codebase
+to a new multi-region approach. In https://github.com/gruntwork-io/terraform-aws-security/releases/tag/v0.51.0[v0.51.0] of
+`terraform-aws-security`, we refactored how we build multi-region modules—that
+is, those modules that deploy resources across every single AWS region, such as `aws-config-multi-region`—to no longer
+create nested provider blocks, and instead, have users pass in providers via the providers map.
+
+Additionally, earlier versions of the account baseline modules did not set the following variables, so please ensure
+that they exist. Here is https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/blob/v0.27.0/examples/for-production/infrastructure-live/logs/_global/account-baseline/terragrunt.hcl#L281[an example] of what you might set the values to for the prod account.
+
+ - `var.config_central_account_id`
+ - `var.security_hub_associate_to_master_account_id`
+ - `var.config_opt_in_regions`
+ - `var.guardduty_opt_in_regions`
+ - `var.kms_cmk_opt_in_regions`
+ - `var.iam_access_analyzer_opt_in_regions`
+ - `var.ebs_opt_in_regions`
+ - `var.security_hub_opt_in_regions`
+ - `var.macie_opt_in_regions`
+
+Once you have completed the above migration steps, it is time to update each baseline module to at least version https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/releases/tag/v0.27.0[v0.27.0] and run Terraform/Terragrunt apply. Typically this is done using the `source` parameter:
```

Comment:
As per two comments above, this seems wrong; the `state mv` won't work properly unless you update to the new version first.